daemon/api_prompting_test.go

```diff
@@ -0,0 +1,652 @@
+// Copyright (c) 2024 Canonical Ltd
```

Comment:
This is missing that `// -*- Mode: Go; indent-tabs-mode: t -*-` header we have in every file. It doesn't bother me but I'm just pointing it out in case its annoying to emacs users

Reply:
changed